glean-core/src/lib.rs

```diff
@@ -689,6 +696,34 @@ impl Glean {
         self.is_first_run
     }
 
+    /// Set a debug view tag.
```

Comment:
Since this is a public API, can you expand a bit on what a "debug view tag" is and what happens to pings tagged this way? You can also link to the docs, if needed.